### PR TITLE
Feature/at tamle 20191024 Activity Intent

### PR DESCRIPTION
**Pull request by at-tamle**
#26

### CODE REVIEW

File: app/src/main/res/drawable/btn_line.xml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
-<shape xmlns:android="http://schemas.android.com/apk/res/android"
+<shape xmlns:android="http://schemas.android.com/apk/res/android">
     android:padding="10dp"
```
Comment (at-dev-support-01) [inline app/src/main/res/drawable/btn_line.xml]: WARNING: [ExtraText] Unexpected text found in layout file: "android:padding="10dp"
    android:shape="rectangle">"
 Layout resource files should only contain elements and attributes. Any XML text content found in the file is likely accidental (and potentially dangerous if the text resembles XML and the developer believes the text to be functional)

File: app/src/main/res/drawable/btn_line.xml [hunk 2]
```diff
@@ -1,6 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
-<shape xmlns:android="http://schemas.android.com/apk/res/android"
+<shape xmlns:android="http://schemas.android.com/apk/res/android">
+    android:padding="10dp"
```
Comment (at-dev-support-01) [inline app/src/main/res/drawable/btn_line.xml]: WARNING: [ExtraText] Unexpected text found in layout file: "android:padding="10dp"
    android:shape="rectangle">"
 Layout resource files should only contain elements and attributes. Any XML text content found in the file is likely accidental (and potentially dangerous if the text resembles XML and the developer believes the text to be functional)

File: app/src/main/res/drawable/btn_line.xml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android">
+    android:padding="10dp"
+    android:shape="rectangle">
```
Comment (at-dev-support-01) [inline app/src/main/res/drawable/btn_line.xml]: WARNING: [ExtraText] Unexpected text found in layout file: "android:padding="10dp"
    android:shape="rectangle">"
 Layout resource files should only contain elements and attributes. Any XML text content found in the file is likely accidental (and potentially dangerous if the text resembles XML and the developer believes the text to be functional)

File: app/src/at-tamle/res/drawable/btn_line.xml [hunk 4]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android">
+    android:padding="10dp"
+    android:shape="rectangle">
```
Comment (at-dev-support-01) [inline app/src/at-tamle/res/drawable/btn_line.xml]: WARNING: [ExtraText] Unexpected text found in layout file: "android:padding="10dp"
    android:shape="rectangle">"
 Layout resource files should only contain elements and attributes. Any XML text content found in the file is likely accidental (and potentially dangerous if the text resembles XML and the developer believes the text to be functional)

File: app/src/at-tamle/java/asiantech/internship/summer/exercise/ProfileActivity.kt [hunk 5]
```diff
@@ -1,13 +1,42 @@
...
 import android.os.Bundle
+import android.provider.MediaStore
+import android.widget.Toast
 import androidx.appcompat.app.AppCompatActivity
 import asiantech.internship.summer.R
+import kotlinx.android.synthetic.`at-tamle`.activity_profile.*
 
 class ProfileActivity : AppCompatActivity() {
-
+    val CAMERA_REQUEST_CODE = 0
```
Comment (at-tainguyen) [inline app/src/at-tamle/java/asiantech/internship/summer/exercise/ProfileActivity.kt]: The final constant should be defined inside the `companion object` block.
In this case, this variant should be marked by `private`

**Pull request merged**